spec/registration_server_spec.rb

```diff
@@ -9,11 +9,13 @@
   let(:port) { 9000 }
   let(:http_server) { double("http_server") }
   let(:certs_factory) { double("certs factory") }
+  let(:updates_url) { URI("http://updates.suse.com/sles12") }
```

Comment:
Now we have these:
```console
$ grep -r '"uri"'
lib/aytests/iso_repo.rb:1:require "uri"
lib/aytests/image_builder.rb:1:require "uri"
spec/servlets/list_updates_spec.rb:3:require "uri"
```
I don't know whether these files are required by the others, I hope the usages of `URI` are covered.